kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/asset/AssetConfiguration.java

```diff
@@ -31,47 +31,6 @@
  * Kura).<br>
  * <br>
  *
- * The properties as provided to an Asset must conform to the following
```

Comment:
I like the idea of not restricting future implementors to follow the current way of using `AssetConfiguration`. Future implementors can use it in different way as well. So, this enhanced and rich javadoc shouldn't be here. But truly this is not efficient way of allowing future implementors to extend the current model. If `AssetConfiguration` is supposed to be extended, we can extract it as an interface and keep its default implementation in Asset Provider. That's the usual way that OSGi Alliance follows. But due to release constraint let's keep it this way.